### DISCUSSION TITLE
Ability to sort remote branches list by recently committed

### DISCUSSION THREAD

**Discussion by peppy** [jesseduffield/lazygit#3018] [Q&A]
Is there a way to do this? Something like `git branch --sort=-committerdate`.

Use case is that I come home and want to quickly find the branches on my own remote which I have just been working on. If I have 200+ branches and can't remember the name, this can be a challenge.

**Comment by stefanhaller** [comment]
No, it's not possible right now, but I want it too. Some related discussion is [here](https://github.com/jesseduffield/lazygit/pull/2147#issuecomment-1656712734) (be sure to click the "load more" button).

More issues that are related: #1412, #1267

**Reply by peppy** [reply]
Thanks, should have searched first.
